xml/snmp_sysdescr.xml

```diff
@@ -1527,28 +1527,65 @@
+      <example os.version="TE1.0.0.175435" hw.series="E20">TANDBERG Codec SoftW: TE1.0.0.175435 MCU: TANDBERG E20 Date: 2009-02-09 S/N: A1AA11B00658 BootSW: Board: 101390-6</example>
       <param pos="0" name="os.certainty" value="0.85"/>
       <param pos="0" name="os.vendor" value="Cisco"/>
       <param pos="0" name="os.product" value="TelePresence"/>
+      <param pos="0" name="os.device" value="Codec"/>
       <param pos="1" name="os.version"/>
-      <param pos="2" name="hw.product"/>
+      <param pos="2" name="hw.series"/>
    </fingerprint>
 
-   <fingerprint pattern="^Cisco TelePresence (Conductor|MCU (?:MSE )?\d+)">
+   <fingerprint pattern="^Cisco TelePresence ((?:HD )?MCU) ((?:MSE )?\d+)">
       <description>Cisco TelePresence</description>
-      <example hw.product="Conductor">Cisco TelePresence Conductor</example>
-      <example hw.product="MCU 4205">Cisco TelePresence MCU 4205</example>
-      <example hw.product="MCU MSE 8420">Cisco TelePresence MCU MSE 8420</example>
+      <example os.device = "MCU" hw.series="4205">Cisco TelePresence MCU 4205</example>
+      <example os.device = "MCU" hw.series="MSE 8420">Cisco TelePresence MCU MSE 8420</example>
+      <example os.device = "HD MCU" hw.series="4520">Cisco TelePresence HD MCU 4520</example>
       <param pos="0" name="os.certainty" value="0.85"/>
       <param pos="0" name="os.vendor" value="Cisco"/>
       <param pos="0" name="os.product" value="TelePresence"/>
-      <param pos="1" name="hw.product"/>
+      <param pos="1" name="os.device"/>
+      <param pos="2" name="hw.series"/>
+   </fingerprint>
+
+   <fingerprint pattern="^Cisco TelePresence (Conductor|Supervisor)(?: (MSE \d+))?">
+      <description>Cisco TelePresence</description>
+      <example os.device="Conductor">Cisco TelePresence Conductor</example>
+      <example os.device="Supervisor" hw.series="MSE 8050">Cisco TelePresence Supervisor MSE 8050</example>
+      <param pos="0" name="os.certainty" value="0.85"/>
+      <param pos="0" name="os.vendor" value="Cisco"/>
+      <param pos="0" name="os.product" value="TelePresence"/>
+      <param pos="1" name="os.device"/>
+      <param pos="2" name="hw.series"/>
+   </fingerprint>
+
+   <fingerprint pattern="^TANDBERG Codec$">
+      <description>Tandberg videoconferencing device</description>
+      <example>TANDBERG Codec</example>
+      <param pos="0" name="os.vendor" value="Cisco"/>
+      <param pos="0" name="os.device" value="Web cam"/>
+   </fingerprint>
+
+   <fingerprint pattern="^TANDBERG MPS-MCU$">
+      <description>Tandberg videoconferencing bridge</description>
+      <example>TANDBERG MPS-MCU</example>
+      <param pos="0" name="os.vendor" value="Cisco"/>
+      <param pos="0" name="os.product" value="MPS-MCU"/>
+      <param pos="0" name="os.device" value="VOIP"/>
```

Comment:
Both in this fingerprint and the one before it, ```os.device``` is using a different pattern than what you are doing elsewhere.